### PR TITLE
Fix first Substrate Chain

### PR DESCRIPTION
**Pull request by shawntabrizi**
* "Runtime" Pallet issues
* Chars per line
* Instance of SRML still there
* Update image

### CODE REVIEW

File: docs/tutorials/creating-your-first-substrate-chain/interact.md [hunk 1]
```diff
@@ -76,13 +73,12 @@ Amount: 1000
...
+If everything went successfully, you should see some notifications in the **Events** component, and
+of course Charlie's balance will now be `1000`.
 
-Already you have a working blockchain, with an underlying cryptocurrency. You
-are able to make transfers easily with a simple, interactive front-end.
+Already you have a working blockchain, with an underlying cryptocurrency. You are able to make
+transfers easily with a simple, interactive front-end.
 
-Now let's build our Proof of Existence Pallet
+Now let's build our Proof of Existence pallet!
```
Comment (riusricardo) [inline docs/tutorials/creating-your-first-substrate-chain/interact.md]: Why are we using `pallet` and not `Pallet`. Isn't Pallet a proper noun? Similar to `Containers`, `Crates`?
Not an important change but important to think about it.

Comment (riusricardo) [inline docs/tutorials/creating-your-first-substrate-chain/interact.md]:
```suggestion
Now let's build our Proof of Existence Pallet!
```

Comment (riusricardo) [reply docs/tutorials/creating-your-first-substrate-chain/interact.md]: Why are we using `pallet` and not `Pallet`. Isn't Pallet a proper noun? Similar to `Containers`, `Crates`?
Not an important change but important to think about it.

Comment (shawntabrizi) [reply docs/tutorials/creating-your-first-substrate-chain/interact.md]: AFAIK no it is not, but would be good to clarify. I will leave it to you to find that answer.

File: docs/tutorials/creating-your-first-substrate-chain/front-end.md [hunk 2]
```diff
@@ -2,18 +2,17 @@
...
-We will give you a custom react component that you can add to your
-`substrate-front-end-template` meant for interacting with your node.
+We will give you a custom react component that you can add to your `substrate-front-end-template`
+meant for interacting with your node.
 
 ## Add Your Custom React Component
 
-In the `substrate-front-end-template` project, edit the `TemplatePallet.js`
-file in the `/src/` folder:
+In the `substrate-front-end-template` project, edit the `TemplateModule.js` file in the `/src/`
```
[the remark this answers is not in the document]
Comment (shawntabrizi) [reply docs/tutorials/creating-your-first-substrate-chain/front-end.md]: ya, if you make the PR on that side, we can update this side.

👌

File: docs/tutorials/creating-your-first-substrate-chain/interact.md (hunk 1 above)
Comment (riusricardo) [reply docs/tutorials/creating-your-first-substrate-chain/interact.md]: I think that you are right. Following my example, `containers` are also just a noun not a proper noun so it is ok to keep it that way.
I was not very clear on my point here. My question should have been related to the title. We sometimes use `Pallet` and not `pallet` in titles.

Comment (shawntabrizi) [reply docs/tutorials/creating-your-first-substrate-chain/interact.md]: In titles we need to follow "Title case"

https://titlecase.com/

Comment (shawntabrizi) [reply docs/tutorials/creating-your-first-substrate-chain/interact.md]: > create a substrate pallet

becomes

> Create a Substrate Pallet

File: docs/tutorials/creating-your-first-substrate-chain/front-end.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JoshOrndorff) [reply docs/tutorials/creating-your-first-substrate-chain/front-end.md]: Proposed plan:
Merge this PR now (or when you're ready)
Cut new node template when v2 is out in which the file renamed to template pallet
Update front end template to match
Revise this article then

[the remark this answers is not in the document]
Comment (shawntabrizi) [reply docs/tutorials/creating-your-first-substrate-chain/front-end.md]: 👌

**Pull request merged**